spec/requests/api/v2/people_controller_create_spec.rb

```diff
@@ -0,0 +1,174 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V2::PeopleController do
+  let(:response_json) { JSON.parse(response.body) }
+  let(:access_token) { create(:access_token).token }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{access_token}") }
+
+  describe 'POST /people' do
+    let(:schema) { load_yaml_schema('post_people_responses.yaml', version: 'v2') }
+
+    let(:ethnicity) { create :ethnicity }
+    let(:gender) { create :gender }
+    let(:risk_type_1) { create :assessment_question, :risk }
+    let(:risk_type_2) { create :assessment_question, :risk }
+
+    let(:person_params) do
+      {
+        data: {
+          type: 'people',
+          attributes: {
+            first_names: 'Bob',
+            last_name: 'Roberts',
+            date_of_birth: Date.civil(1980, 1, 1),
+            prison_number: 'G3239GV',
+            criminal_records_office: 'CRO0111d',
+            police_national_computer: 'AB/1234567',
+            gender_additional_information: 'info about Bob',
+          },
+          relationships: {
+            ethnicity: {
+              data: {
+                id: ethnicity.id,
+                type: 'ethnicities',
+              },
+            },
+            gender: {
+              data: {
+                id: gender.id,
+                type: 'genders',
+              },
+            },
+          },
+        },
+      }
+    end
+
+    context 'when successful' do
```

Comment:
I don't think successful describes anything/adds any value?

Reply:
I know I hate it, it's just legacy stuff, that's not even a proper contest for me,  I agree, I am going to remove it.

Reply:
Yay \0/